CHANGELOG

```diff
@@ -0,0 +1,107 @@
+==== 2.1 beta 2017-12-22 ====
+After two years of hard work, we are pleased to announce Mixxx 2.1 beta. We
```

Comment:
nit-picking: a line break after "...beta."

Reply:
Most people will be reading this on a webpage, so word wrapping does not matter.

Reply:
Ah okay..